api/migrations/20181120160446_add-contractor-total-cost.js

```diff
@@ -0,0 +1,13 @@
+exports.up = async knex => {
+  await knex.schema.alterTable('activity_contractor_resources', table => {
+    table
+      .decimal('totalCost', 15, 2)
```

Comment:
Normally I do column names in snake_case, but this table already had some columns that were camelCased, so rather than change those or introduce inconsistency within the table, I made this one camelCase too.  It'd probably be a worthwhile exercise to pick one and update the entire schema accordingly.  Probably camelCase, since that'd eliminate a lot of the data massaging that happens in the data model.